src/deno_dir.rs

```diff
@@ -136,38 +137,78 @@ impl DenoDir {
   }
 
   // Prototype https://github.com/denoland/deno/blob/golang/deno_dir.go#L37-L73
+  /// Fetch remote source code.
   fn fetch_remote_source(
     self: &Self,
     module_name: &str,
     filename: &str,
-  ) -> DenoResult<(String, msg::MediaType)> {
-    let p = Path::new(filename);
-    // We write a special ".mime" file into the `.deno/deps` directory along side the
-    // cached file, containing just the media type.
-    let mut media_type_filename = filename.to_string();
-    media_type_filename.push_str(".mime");
-    let mt = Path::new(&media_type_filename);
-
-    let src = if self.reload || !p.exists() {
-      eprintln!("Downloading {}", module_name);
-      let (source, content_type) = http_util::fetch_sync_string(module_name)?;
-      match p.parent() {
-        Some(ref parent) => fs::create_dir_all(parent),
-        None => Ok(()),
-      }?;
-      deno_fs::write_file(&p, source.as_bytes(), 0o666)?;
-      deno_fs::write_file(&mt, content_type.as_bytes(), 0o666)?;
-      (source, map_content_type(&p, Some(&content_type)))
-    } else {
-      let source = fs::read_to_string(&p)?;
-      // .mime file might not exists with bundled deps
+  ) -> DenoResult<Option<CodeFetchOutput>> {
+    let extensions = ["", ".ts", ".js"];
+    for ext in extensions.iter() {
+      let filename = format!("{}{}", filename, ext);
+      let module_name = format!("{}{}", module_name, ext);
+      let p = Path::new(&filename);
+      // We write a special ".mime" file into the `.deno/deps` directory along side the
+      // cached file, containing just the media type.
+      let mut media_type_filename = filename.to_string();
+      media_type_filename.push_str(".mime");
+      let mt = Path::new(&media_type_filename);
+      eprintln!("Try downloading {}...", &module_name);
```

Comment:
Please keep it "Downloading"

Reply:
I have actually made some changes to this just now. Now it looks like
```sh
Downloading https://deno.land/x/net/file_server.ts... DONE
Compiling https://deno.land/x/net/file_server.ts
Downloading https://deno.land/x/net/http... NOT FOUND
Downloading https://deno.land/x/net/http.ts... DONE
Downloading https://deno.land/x/net/bufio.ts... DONE
Downloading https://deno.land/x/net/textproto.ts... DONE
Downloading https://deno.land/x/net/http_status... NOT FOUND
Downloading https://deno.land/x/net/http_status.ts... DONE
Downloading https://deno.land/x/net/util... NOT FOUND
Downloading https://deno.land/x/net/util.ts... DONE
Compiling https://deno.land/x/net/http.ts
...
HTTP server listening on http://0.0.0.0:4500/
```

Reply:
I'd rather you keep it as a single line - in the case of an error it can be printed too. The reason is that I want to refactor this whole bit soon with progress bars - so I prefer if you keep it the same and as simple as possible for future refactors.

Reply:
Oh okay. Done.